templates/index.html

```diff
@@ -49,6 +49,7 @@ <h1>
 <br />
 <div class="row">
 {{ blogger_html|safe }}
+{{ bluesky_html|safe }}
```

Comment:
I'm guessing this doesn't actually work, since we don't have an HTML template here to get the handle and app password? I assume we should remove this then?

Reply:
Ah yeah I'll drop it